vue-suggestions.vue

```diff
@@ -3,7 +3,7 @@
 </template>
 
 <script>
-  import $ from 'jquery';
+  import jQuery from 'jquery';
```

Comment:
Why do you change $ to jQuery?

Reply:
I was trying to find where actually you use jQuery. My editor had said me that `this.$el` is jQuery also :smile_cat:

Reply:
What editor do you use? Please, revert to $.

Reply:
You can do it with any editor with "Find and replace with chr" function.

`$` is the alias of jQuery. I prefer to use full name. That is much simpler to read especially when we have `$` (jQuery) and another tool like angular/Vue with internal `$` notation. You need concentrate and think carefully about each like, see on the context of usage or just use full name (`jQuery`) and read the code quickly. I prefer second approach. Because it saves my time, prevents mistakes (bugs), etc.

I've done requested changes because it's your project but I don't agree with this one =)